addons/website_event/views/event_templates_list.xml

```diff
@@ -149,9 +149,20 @@
     <!-- No events -->
     <t t-if="not event_ids">
         <div class="col-12">
-            <div class="h2 mb-3">No events found.</div>
-            <div class="alert alert-info text-center" groups="event.group_event_user">
-                <p class="m-0">Use the top button '<b>+ New</b>' to create an event.</p>
+            <div class="d-flex flex-column align-items-center justify-content-center text-center">
+                <div class="mb-3">
+                    <t t-call="website_event.event_svg"/>
+                </div>
+                <h2 class="mb-2">No events scheduled yet</h2>
+                <p>We couldn’t find any events scheduled at this moment.</p>
```

Comment:
Can we maybe do a t-if/t-else to say "we couldn't find any event matching your search criteria"? 

This way it's more accurate, and we're consistent with the other messages. We should have access to ``searches['search']`` here.

Reply:
Updated, can you check if it was done the right way?